docs/tutorials/add-lists.md

```diff
@@ -31,7 +32,7 @@ module.exports = {
 };
 ```
 
-Here we described a very basic schema of generic ToDo. Let's register it.
+Here we described a very basic schema for a generic `Todo`. Let's add it to our Keystone system.
```

Comment:
```suggestion
Here we described a very basic schema for a generic `Todo`. Let's add it to our Keystone application.
```

Is application better than system (don't mind if you have reasons to prefer system)